Docs: experimental CLI docs (#17)
Unleash/unleash-client-nextjs: README.md

};
```

# ⚗️ CLI (experimental)

You can use `unleash [action] [options]` in your `package.json` `scripts` section, or with:

```sh
npx @unleash/nextjs
```

For the CLI to work, the following [environment variables](#environment-variables) must be configured with appropriate values:

- `UNLEASH_SERVER_API_URL`
- `UNLEASH_SERVER_API_TOKEN`

> The CLI will attempt to environment values from any `.env` files if they're present. You can also set the variables directly when invoking the interface, as in the [CLI usage example](#example).

Reviewer: Realized this was missing a word 😅 

```suggestion
The CLI will attempt to read environment values from any `.env` files if they're present. You can also set the variables directly when invoking the interface, as in the [CLI usage example](#example).
```

Author: I think I will drop shorthand ENV for simplicity.

};
```

# ⚗️ CLI (experimental)

You can use `unleash [action] [options]` in your `package.json` `scripts` section, or with:

```sh
npx @unleash/nextjs
```

> CLI will attemp to read NextJS `.env` files. You need to set `UNLEASH_SERVER_API_URL` and `UNLEASH_SERVER_API_TOKEN` (see [environment variables](#environment-variables)). Another way of configuring the source of feature toggles is to pass environment variables directly, like in the example below. Available shorthand variables are: `UNLEASH_URL`, `UNLEASH_TOKEN`, `UNLEASH_APPNAME`.

Reviewer: I'm still not sure **why** I need to set these environment variables. What do they do? 

And also, what are the shorthand variables for? They're not mentioned in the "environment variables" section. Why not? Are they only for the CLI? 

`UNLEASH_APPNAME` is also different from `UNLEASH_APP_NAME`. Is that intentional or a typo?

```suggestion
For the CLI to work, the following [environment variables](#environment-variables) must be configured with appropriate values:

- `UNLEASH_SERVER_API_URL` 
- `UNLEASH_SERVER_API_TOKEN`

The CLI will attempt to environment values from any `.env` files if they're present. You can also set the variables directly when invoking the interface, as in the [CLI usage example](#example).
```

<br />

# ⚗️ CLI (experimental)

> You can use `unleash [action] [options]` in you `package.json` `scripts` section, or with:

Reviewer: ```suggestion
You can use `unleash [action] [options]` in your `package.json` `scripts` section, or with:
```

```sh
npx @unleash/nextjs
```

> Use [Environment Variables](#environment-variables) (you needed: `UNLEASH_SERVER_API_URL`, `UNLEASH_SERVER_API_TOKEN`), or shorthand versions: `UNLEASH_URL`, `UNLEASH_TOKEN`, `UNLEASH_APPNAME`.

Reviewer: What do you use environment variables for? I don't understand why this sentence is telling me to do something.

```suggestion
Use [environment variables](#environment-variables) (you need: `UNLEASH_SERVER_API_URL`, `UNLEASH_SERVER_API_TOKEN` or shorthand versions: `UNLEASH_URL`, `UNLEASH_TOKEN`, `UNLEASH_APPNAME`).
```

Author: Updated

## Usage

- `get-definitions <outputFile.json>` Download feature flags definitions for bootstrapping (offline use) of server-side SDK.
- `generate-types [options] <outputFile.ts>` Generate types and typed functions from feature flags defined in an Unleash instance.
  It will also generate strictly typed versions of `useFlag`, `useVariant`, `useFlags` and `flagsClient` (unless `--typesOnly` is used).
  - `-t, --typesOnly` don't include typed versions of functions exported from `@unleash/nextjs` (default: false)
  - `-b, --bootstrap <sourceFile.json>` load definitions from a file instead of fetching definitions - work offline
- `-V` Output the version number

## Example

Try it now

```sh
UNLEASH_URL=https://app.unleash-hosted.com/demo/api \
UNLEASH_TOKEN=test-server:default.8a090f30679be7254af997864d66b86e44dcfc5291916adff72a0fb5 \
npx @unleash/nextjs generate-types ./unleash.ts
```

> <br/>

Reviewer: I let a few of these slip through previously, but I personally don't like using manual breaks in the markup. Much like with HTML, I don't think the text should try and define how it's presented. So I suggest we remove this one. 

That said, I'm more than happy to let you make a case for why we should keep them if you think we should.

```suggestion
```

Reviewer: @Tymek I noticed you resolved this one and removed all the breaks. But it seems you might have forgotten this one? 😅
